public/app/features/alerting/unified/components/rules/EditRuleGroupModal.tsx

```diff
@@ -99,9 +102,30 @@ export function EditCloudGroupModal(props: Props): React.ReactElement {
                 placeholder="1m"
                 {...register('groupInterval', {
                   pattern: durationValidationPattern,
+                  validate: (input) => {
+                    const validDuration = isValidGoDuration(input);
+                    if (!validDuration) {
+                      return 'Invalid duration. Valid example: 1m (Available units: h, m, s)';
+                    }
+
+                    const limitExceeded = !checkEvaluationIntervalGlobalLimit(input).exceedsLimit;
+                    if (limitExceeded) {
+                      return true;
+                    }
+
+                    return false;
+                  },
                 })}
               />
             </Field>
+            {checkEvaluationIntervalGlobalLimit(watch('groupInterval')).exceedsLimit && (
+              <Alert severity="warning" title="Global evalutation interval limit exceeded">
```

Comment:
Should we extract this to a separate component and use also in the `GrafanaEvaluationBehavior` file?